frontend/server/src/Controllers/Course.php

```diff
@@ -762,14 +762,17 @@ private static function createAssignment(
 
             if (!empty($addedProblems)) {
                 foreach ($addedProblems as $i => $addedProblem) {
+                    $commit = isset(
+                        $addedProblem['commit']
+                    ) ? $addedProblem['commit'] : null;
```

Comment:
```suggestion
                    $commit = $addedProblem['commit'] ?? null,
```

esto funciona? si sí, igual y conviene darle inline en L775

Reply:
Debería funcionar, pero no recuerdo si psalm se estaba quejando. Pudo ser error mío, ya quedó.